docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md

```diff
@@ -0,0 +1,90 @@
+---
+description: XNCP Basics, using 010 Editor
+---
+!!! info
+    This guide uses the following tools:
+
+    - 010 Editor (with XNCP Template)
+
+    These tools and more can be found in the [XNCP tools page](/tools/hedgehog-engine/blueblur/files/xncp)
+# The Basics of XNCP Editing
```

Comment:
Have you tested this locally btw? Not sure if you need an extra line between the info box and the page title...

Reply:
I have tested it locally, with or without the space between info and title, it looks the exact same!

Reply:
![image](https://user-images.githubusercontent.com/38560522/163812703-60814384-6b0b-4bd0-a09b-cfd9ee33faf7.png)

Reply:
Nice so it's working properly. Seems good, will merge